### ISSUE TITLE
[HOW TO]预训练参数加载

### ISSUE THREAD

**Issue by AIHHU** [PaddlePaddle/PaddleClas#1863]
你好，请问，我做迁移学习的时候，我只想导入特征提取层的预训练参数，但是我不想导入输出层（比如:softmax）的预训练参数，因为我要换掉该层，请问应该怎么做呢？

**Comment by HydrogenSulfate** [comment]
可以基于你的训练参数文件重新生成一份没有输出层，只有特征提取层的参数然后保存下来，然后把这份文件的路径写到yaml文件的Arch.backbone.pretrained后面
